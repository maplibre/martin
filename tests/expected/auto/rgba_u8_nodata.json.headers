content-encoding: br
content-type: application/json
etag: W/"69-f7ZqJowAoXXdyYxlTrEd6g=="
transfer-encoding: chunked
vary: accept-encoding, Origin, Access-Control-Request-Method, Access-Control-Request-Headers
