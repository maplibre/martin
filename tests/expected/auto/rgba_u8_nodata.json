{
  "maxzoom": 2,
  "minzoom": 0,
  "tilejson": "3.0.0",
  "tiles": [
    "http://localhost:3111/rgba_u8_nodata/{z}/{x}/{y}"
  ]
}
